### PR TITLE
Emit a message for each additional tag created

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @smarterclayton PTAL

### CODE REVIEW

File: dockerclient/client.go [hunk 1]
```diff
@@ -344,6 +344,7 @@ func (e *ClientExecutor) Commit(b *imagebuilder.Builder) error {
 				e.Deferred = append(e.Deferred, func() error { return e.Client.RemoveImageExtended(image.ID, docker.RemoveImageOptions{Force: true}) })
 				return fmt.Errorf("unable to tag %q: %v", s, err)
 			}
+			e.LogFn("Committing changes to %s ...", tag)
```
Comment (smarterclayton) [inline dockerclient/client.go]: Message should be "Tagging as %s"

**Comment by stevekuznetsov** [comment]
@smarterclayton updated

File: dockerclient/client.go [hunk 2]
```diff
@@ -344,6 +344,7 @@ func (e *ClientExecutor) Commit(b *imagebuilder.Builder) error {
 				e.Deferred = append(e.Deferred, func() error { return e.Client.RemoveImageExtended(image.ID, docker.RemoveImageOptions{Force: true}) })
 				return fmt.Errorf("unable to tag %q: %v", s, err)
 			}
+			e.LogFn("Tagging as %s ...", tag)
```
Comment (smarterclayton) [inline dockerclient/client.go]: Doesn't need the ..., it's not in progress ;)

Comment (smarterclayton) [inline dockerclient/client.go]: By the time we've reached here, we've already tagged.

**Comment by stevekuznetsov** [comment]
@smarterclayton updated verb tense

**Pull request merged**